src/components/application_manager/src/commands/mobile/get_vehicle_data_request.cc

```diff
@@ -268,28 +268,46 @@ void GetVehicleDataRequest::on_event(const event_engine::Event& event) {
   switch (event.id()) {
     case hmi_apis::FunctionID::VehicleInfo_GetVehicleData: {
       EndAwaitForInterface(HmiInterfaces::HMI_INTERFACE_VehicleInfo);
+
+      smart_objects::SmartObject incoming_msg_params = message[strings::msg_params];
+      
+      if (incoming_msg_params.keyExists(strings::gps)) {
+          if (incoming_msg_params[strings::gps].empty()) {
```

Comment:
@KVGrygoriev Why not to write : 
```
 if (incoming_msg_params.keyExists(strings::gps) && incoming_msg_params[strings::gps].empty())
```
instead of nested `if()` statements?

Reply:
@AByzhynar , see 41aa1c3b2744ac4a3d72c905a500c5b7b1e45ab3